push/Push.cpp

```diff
@@ -226,7 +226,7 @@ std::string CPush::ProcessSendValue(const std::string &rawsendValue, const int d
 	}	
 	else if (vType == "Counter")
 	{
-		sprintf(szData,"Not supported yet");
+		sprintf(szData,"%d", atoi(rawsendValue.c_str()));
```

Comment:
Sure np. I'm not that familiar with this but why not `strtoul` or `strtoull`? (`strtol` instead of `atoi`?)